packages/ui/src/services/color/ColorCSSVariablesUpdater.stories.ts

```diff
@@ -1,12 +1,36 @@
-import { defineComponent } from 'vue'
-import ColorCSSVariablesUpdater from './ColorCSSVariablesUpdater.demo.vue'
+import { VaChip, VaButton } from '../../components'
+import { useColors } from '../../composables'
 
 export default {
-  title: 'ColorCSSVariablesUpdater',
-  component: ColorCSSVariablesUpdater,
+  title: 'services/ColorCSSVariablesUpdater',
 }
 
-export const Default = defineComponent({
-  components: { ColorCSSVariablesUpdater },
-  template: '<ColorCSSVariablesUpdater/>',
+export const Default = () => ({
+  components: { VaChip, VaButton },
+  setup () {
+    const colors = useColors()
+    const change = (color) => {
+      colors.setColors({ danger: color })
+    }
+
+    return {
+      change,
+    }
+  },
+  template: `
+    <div class="mb-6">
+      <a style="color: var(--va-danger)">
+        This is title danger color
+      </a>
+      <va-chip color="danger">This is vue component</va-chip>
```

Comment:
Also, this story tests two cases right now: css variable and how component reacts to color config update. Should only test css variables here and test second case somewhere else. (Idk where).

```suggestion
```